pkg/runner/dochandlers/v23/v23.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package v23
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/opensbom-generator/parsers/meta"
+	"github.com/spdx/spdx-sbom-generator/pkg/runner/options"
+	"github.com/spdx/tools-golang/spdx"
+)
+
+type Handler struct{}
+
+func (h *Handler) CreateDocument(opts *options.Options) (interface{}, error) {
+	doc := &spdx.Document{
+		SPDXVersion: spdx.Version,
+		DataLicense: spdx.DataLicense,
+		// ... this needs to be completed with all the required fuields
+	}
+
+	// TODO: Create top-level package
+	// TODO: Relate top-level package to document
```

Comment:
@ba11b0y:

Thew SPDX document needs to have a top-level package representing the source code base. All the packages coming from the parsers should be related to that top-level element (not directly to the document).